tests/unit_tests/test_meta.cpp

```diff
@@ -260,14 +260,14 @@ namespace gridtools {
                           f<void, void, void, int, double, void, void>>::value,
             "");
 
-        // void_t (CWG 1558 https://wg21.cmeerw.net/cwg/issue1558)
+        // std::void_t (CWG 1558 https://wg21.cmeerw.net/cwg/issue1558)
         namespace defect_cwg_1558 {
```

Comment:
please remove the whole namespace -- it doesn't test any `meta` functionality now